docs/resources/Channel.md

```diff
@@ -6,41 +6,41 @@ Represents a guild or DM channel within Discord.
+| recipients?                         | array of [user](#DOCS_RESOURCES_USER/user-object) objects                   | the recipients of the DM                                                                                                                                                                      | DM, GROUP_DM                                                                                                                                                       |
+| icon?                               | ?string                                                                     | icon hash of the group DM                                                                                                                                                                     | GROUP_DM                                                                                                                                                           |
+| owner_id?                           | snowflake                                                                   | id of the creator of the group DM or thread                                                                                                                                                   | GROUP_DM, ANNOUNCEMENT_THREAD, PUBLIC_THREAD, PRIVATE_THREAD                                                                                                       |
+| application_id?                     | snowflake                                                                   | application id of the group DM creator if it is bot-created                                                                                                                                   | GROUP_DM                                                                                                                                                           |
+| parent_id?                          | ?snowflake                                                                  | for guild channels: id of the parent category for a channel (each parent category can contain up to 50 channels), for threads: id of the text channel this thread was created                 | DM, GUILD_VOICE, GUILD_CATEGORY, GUILD_ANNOUNCEMENT, ANNOUNCEMENT_THREAD, PUBLIC_THREAD, PRIVATE_THREAD, GUILD_STAGE_VOICE, FORUM_CHANNEL                          |
+| last_pin_timestamp?                 | ?ISO8601 timestamp                                                          | when the last pinned message was pinned. This may be `null` in events such as `GUILD_CREATE` when a message is not pinned.                                                                    | DM, GUILD_ANNOUNCEMENT, FORUM_CHANNEL                                                                                                                              |
```

Comment:
```suggestion
| last_pin_timestamp?                 | ?ISO8601 timestamp                                                          | when the last pinned message was pinned. This may be `null` in events such as `GUILD_CREATE` when a message is not pinned.                                                                    | DM, GUILD_ANNOUNCEMENT, FORUM_CHANNEL, GUILD_TEXT, PUBLIC_THREAD, PRIVATE_THREAD                                                                                                                            |
```
those have pins too

Reply:
What has pins too?